### ISSUE TITLE
Module not found:

### ISSUE THREAD

**Issue by eeeman1** [Flipboard/react-canvas#170]
Module not found: Error: Can't resolve 'react-dom/lib/ReactMultiChild' in '.../node_modules/react-canvas/lib'

**Comment by rardoz** [comment]
Yeah I had that too. I switched to react-konva. The issue is because it needs react-dom@15.x

**Comment by scoutrul** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by hugohuang1111** [comment]
+1

**Comment by riseremi** [comment]
+1

**Comment by ghost** [comment]
项目太长时间没有维护了，**react-dom**内部一些模块早已发生了变化，已致于无法引入**react-dom/lib/ReactMultiChild**。

The project has not been updated a long time ago. and some modules inside the **react-dom** have already changed, the **react-dom/lib/ReactMultiChild** also changed.

**Comment by TaeWoongKim2** [comment]
Thanks for your clear answer!
